apps/backend/src/frameworks/casl/app.roles.ts

```diff
@@ -0,0 +1,5 @@
+export enum Roles {
+  Admin = 'admin',
+  Owner = 'owner',
+  User = 'user',
+}
```

Comment:
Why is having this distinction between User/Owner relevant?

Reply:
Basically, an owner can't adopt pets. By default, every new joiner would be considered a user. Once they post adoptions, they become owners. IMO, a user who is posting pets for adoption can't adopt other pets. Otherwise, we might leave an open door for pet exchange, which is not what we want.

Reply:
@belgamo I'd call that out in this module for future reference. I think adding some documentation at the top would be helpful for onboarding new folks in this project